### PR TITLE
[Datahub] Add pagination for organisations component

### PR DESCRIPTION
**Pull request by tkohr**
The PR implements pagination for the `organisations.component`, using the UI `pagination.component` as illustrated on the screenshot below:

pagination

### CODE REVIEW

File: libs/ui/elements/src/lib/pagination/pagination.component.html [hunk 1]
```diff
@@ -2,17 +2,17 @@
   <button
     (click)="nextPage()"
     [disabled]="currentPage === nPages"
-    class="uppercase text-white flex justify-center items-center p-3 rounded-lg mr-8 cursor-pointer h-14 mr-12 bg-[#8cc630] disabled:bg-gray-100 disabled:cursor-default"
+    class="uppercase text-white justify-center items-center p-3 rounded-lg cursor-pointer h-14 w-44 bg-secondary hidden sm:flex sm:mr-28 disabled:bg-gray-100 disabled:cursor-default"
```
Comment (jahow) [inline libs/ui/elements/src/lib/pagination/pagination.component.html:5]: Just a remark on the `sm:` classes: I think this is a bit too "context-dependant", as the screen width is not really a strong indication that there is room for the button. To keep in mind if this component has to be reused somewhere!

File: libs/ui/elements/src/lib/pagination/pagination.component.html [hunk 2]
```diff
@@ -2,17 +2,17 @@
...
   <div class="flex items-center" translate>
-    <span class="m-3" translate> datahub.search.page </span>
-    <span class="border border-gray-300 rounded py-1 px-5 mr-3">{{
+    <span class="m-3" translate> pagination.page </span>
+    <span class="border border-gray-300 rounded w-14 py-1 px-5 mr-3">{{
       currentPage
     }}</span>
-    <span class="mr-3" translate> datahub.search.pageOn </span>
+    <span class="mr-3" translate> pagination.pageOf </span>
     <span class="mr-3">{{ nPages }}</span>
```
Comment (jahow) [inline libs/ui/elements/src/lib/pagination/pagination.component.html:16]: Would it be possible to combine this into one translation key with the total page count as parameter?

**Review by jahow — APPROVED** [review]
Thanks, good work! I've added a couple of remarks, but nothing mandatory for this PR.

File: libs/ui/elements/src/lib/pagination/pagination.component.html (hunk 1 above)
Comment (tkohr) [reply libs/ui/elements/src/lib/pagination/pagination.component.html:5]: Ok, I see your point. What do you recommend for the case that the screen becomes too narrow to display the "next page" button on the left to the pagination buttons ? Letting the pagination buttons wrap below doesn't really look neat IMO.

File: libs/ui/elements/src/lib/pagination/pagination.component.html (hunk 2 above)
Comment (tkohr) [reply libs/ui/elements/src/lib/pagination/pagination.component.html:16]: Yes indeed, tested it quickly with `[translateParams]="{ pages: nPages }"`, but will keep as is for now as the count param displays as `undefined` during load time when passing it to the translation.

**Comment by tkohr** [comment]
Thanks for the review @jahow !

File: libs/ui/elements/src/lib/pagination/pagination.component.html (hunk 1 above)
Comment (jahow) [reply libs/ui/elements/src/lib/pagination/pagination.component.html:5]: An endless problem... soon [CSS will allow handling sizes at the block level](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries) and not just the page, but for now there is no straightforward way to do this.

A solution might be to leave the button there but make sure that it correctly repositions if there is not enough room, using flex.

**Pull request merged**